data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/SchemaConfig.java

```diff
@@ -37,6 +43,51 @@ public class SchemaConfig {
   @JsonProperty("basic_auth_credentials_source")
   private String basicAuthCredentialsSource;
 
+  @JsonProperty("inline_schema")
+  private String inlineSchema;
+
+  @JsonProperty("schema_file_location")
+  private String schemaFileLocation;
+
+  @JsonProperty("s3_file_config")
+  private S3FileConfig s3FileConfig;
+
+  @JsonProperty("is_create")
```

Comment:
not sure what does is_create mean? create topic? should be renamed as per functionality